app/lib/server/startup/settings.js

```diff
@@ -1283,10 +1283,18 @@ settings.addGroup('Push', function() {
 			type: 'boolean',
 			public: true,
 		});
-		return this.add('Push_show_message', true, {
+		this.add('Push_show_message', true, {
 			type: 'boolean',
 			public: true,
 		});
+		return this.add('Push_request_content_from_server', true, {
```

Comment:
we can get rid of that `return` 🙈 
```suggestion
	this.add('Push_request_content_from_server', true, {
```